libbeat/processors/actions/decode_json_fields.go

```diff
@@ -122,6 +122,10 @@ func (f *decodeJSONFields) Run(event *beat.Event) (*beat.Event, error) {
 		if err != nil {
 			f.logger.Debugf("Error trying to unmarshal %s", text)
 			errs = append(errs, err.Error())
+			event.SetErrorWithOption(common.MapStr{
+				"message": "parsing input as JSON: " + err.Error(),
+				"input":   text,
+			}, f.addErrorKey)
```

Comment:
@kvch Is there any standard structure for this error field? What do you think about this implementation?

Reply:
This looks good. I would also add the name of the field that contains the invalid JSON.